Build fails in MSVC

14>pch.obj : error LNK2005: "private: static int const Background::Starfield::BG_STAR_MAX" (?BG_STAR_MAX@Starfield@Background@@0HB) already defined in Background.obj
14>pch.obj : error LNK2005: "private: static int const Background::Starfield::BG_STAR_MIN" (?BG_STAR_MIN@Starfield@Background@@0HB) already defined in Background.obj
14>  exception_handler.lib(exception_handler.obj) : MSIL .netmodule or module compiled with /GL found; restarting link with /LTCG; add /LTCG to the link command line to improve linker performance
14>pch.obj : error LNK2005: "private: static int const Background::Starfield::BG_STAR_MAX" (?BG_STAR_MAX@Starfield@Background@@0HB) already defined in Background.obj
14>pch.obj : error LNK2005: "private: static int const Background::Starfield::BG_STAR_MIN" (?BG_STAR_MIN@Starfield@Background@@0HB) already defined in Background.obj
14>LINK : warning LNK4098: defaultlib 'LIBCMT' conflicts with use of other libs; use /NODEFAULTLIB:library
14>I:\github\pioneer\win32\vc2013Express\..\..\pioneer.exe : fatal error LNK1169: one or more multiply defined symbols found

--- a/src/Background.cpp
+++ b/src/Background.cpp
@@ -204,9 +204,6 @@ void Starfield::Init()
 	m_material->emissive = Color::WHITE;
 }
 
-const int Starfield::BG_STAR_MAX;
-const int Starfield::BG_STAR_MIN;
-
 void Starfield::Fill(Random &rand)
 {
 	const Sint32 NUM_BG_STARS = Clamp(Sint32(Pi::GetAmountBackgroundStars() * BG_STAR_MAX), BG_STAR_MIN, BG_STAR_MAX);

--- a/src/Background.h
+++ b/src/Background.h
@@ -60,8 +60,8 @@ namespace Background
 
 	private:
 		void Init();
-		static const int BG_STAR_MAX = 100000;
-		static const int BG_STAR_MIN = 1000;
+		#define BG_STAR_MAX 100000
+		#define BG_STAR_MIN 1000
 		std::unique_ptr<Graphics::VertexBuffer> m_vertexBuffer;
 
 		//hyperspace animation vertex data

